src/components/masters/program/ProgramPage.jsx

```diff
@@ -135,7 +141,16 @@ ProgramPage.propTypes = {
   pageContext: PropTypes.shape({
     programName: PropTypes.string.isRequired,
     programSlug: PropTypes.string.isRequired,
-    programUUID: PropTypes.string.isRequired,
+    programUUID: PropTypes.shape({
```

Comment:
I think you switched `programUUID` and `programBranding`?

Reply:
👍 Fixed it and added a shape to branding.